features/step_definitions/web_steps.rb

```diff
@@ -0,0 +1,55 @@
+Given(/^I visit the homepage$/) do
+  visit('/')
+end
+
+When(/^I follow "(.*?)"$/) do |link|
+  click_link(link)
+end
+
+Then(/^I see a "(.*?)" message$/) do |message|
+  expect(page).to have_content(message)
+end
+
+
+Then(/^I see a blank board$/) do
+  expect(page).to have_content('0')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.